modules/core/src/main/java/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -439,7 +439,6 @@ public String handleResponse(final ClassicHttpResponse response) throws IOExcept
                             calculateProgressPercent(update.totalExpectedSize, update.totalRetrievedSize));
 
                     JSONUtilities.safePut(fileRecord, "size", saveStreamToFile(stream, file, null));
-                    JSONUtilities.safePut(fileRecord, "format", guessBetterFormat(file, request.getCharacterEncoding(), "text"));
```

Comment:
This is not the only place where a `"format"` is set in this method (see the cases above for `"clipboard"` and `"download"` cases). So by removing it only in this particular case we are introducing a discrepancy.

Reply:
Flow updated - format identification using the newly added method is used for clipboard, url/download and files in the import flow.